templates/github/.github/workflows/nightly.yml.j2

```diff
@@ -109,20 +112,20 @@ jobs:
           tar xvf "../../{{ plugin_name }}/{{ plugin.app_label }}-python-client.tar"
           popd
           {%- endfor %}
+      {%- endif %}
 
-      {{ setup_python() | indent(6) }}
+      {{ install_python_deps(["towncrier", "twine", "wheel", "httpie", "docker", "netaddr", "boto3", "ansible~=10.3.0", "mkdocs", "jq", "jsonpatch", "bump-my-version"]) | indent(6) }}
 
       {{ setup_env(test="performance") | indent(6) }}
 
-      {{ install_python_deps(["towncrier", "twine", "wheel", "httpie", "docker", "netaddr", "boto3", "ansible", "mkdocs"]) | indent(6) }}
-
       {{ run_script(name="Before Install", file="before_install.sh") | indent(6) }}
 
       {{ run_script(name="Install", file="install.sh") | indent(6) }}
 
       {{ run_script(name="Before Script", file="before_script.sh", extra_env={"REDIS_DISABLED": "${{ contains('%s', matrix.env.TEST) }}" % (disabled_redis_runners | default(["TILT"]) | join(' '))}) | indent(6) }}
 
-      {{ run_script(name="Performance Test", file="script.sh") | indent(6) }}
+      {{ run_script(name="Script", file="script.sh") | indent(6) }}
 
       {{ display_logs() | indent(6) }}
 {%- endif %}
+...
```

Comment:
Why this `...` ?

Reply:
This is being polite in yaml.

Reply:
I'm also on a crusade quote all string values.

Reply:
I'm following the quote crusade, but i was not aware of the ... thing.

Reply:
Oh yeah, its usually used in multifile yaml. I remembered it as was just `---`.